### ISSUE TITLE
test: `cheat.bats` is testing `stocks`

### ISSUE THREAD

**Issue by virgilwashere** [alexanderepstein/Bash-Snippets#184]
**Issue Label:**
* [x] Bug
* [ ] New feature
* [ ] Enhancement
* [ ] New component

**Description:**

[`cheat.bats`](tests/cheat.bats) was overwritten with [`stocks.bats`](tests/stocks.bats) a while back.

### If its a bug make sure to include this section.
**OS and OS version:**
* [ ] Mac
* [ ] Linux 32 Bit
* [x] Linux 64 Bit
* [ ] Windows 32 Bit
* [ ] Windows 64 Bit

OS Version:
```console
$ uname -a
Linux firefly 4.4.0-18362-Microsoft #1-Microsoft Mon Mar 18 12:02:00 PST 2019 x86_64 x86_64 x86_64 GNU/Linux
$ lsb_release -ds
Ubuntu 18.04.1 LTS
```

**Comment by alexanderepstein** [comment]
Fixed in #185

**Comment by alexanderepstein** [comment]
@virgilwashere So it seems most tests are now working on OSX except for a single weather test if you happen to get time to fix https://travis-ci.org/alexanderepstein/Bash-Snippets/jobs/528642485 . 

Additionally if possible I would like to fix the Linux tests so that both systems can be part of the required CI pipeline. 

If you are busy or don't want to do these things that is alright as well, I appreciate what you have already done!
